kentikapi/client.go

```diff
@@ -3,11 +3,11 @@ package kentikapi
 import (
```

Comment:
generate_server.sh  generates the code to wrong directory. Please fix that.
Also, git apply command in the script fails because of bad path. If you fix the path it will probably still fails, because patch itself contains paths. You could try modifying the paths in patch by hand to fix that.

Reply:
Script fails with `error: can't open patch './kentikapi/fakeserver/patches/0001-Modify-fakeserver-not-to-require-preset-in-.patch': No such file or directory
`